src/index.html

```diff
@@ -9,6 +9,16 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="container">
+      <div class="clock">
+        <div class="clock__face">
+          <div class="clock__handles">
```

Comment:
rename to `hands`/`hand`
```suggestion
          <div class="clock__hands">
```